package.json

```diff
@@ -8,22 +8,30 @@
     "eslint-plugin"
   ],
   "engines": {
-    "node": ">=0.10.0"
+    "node": ">=12.22.0"
   },
   "license": "ISC",
-  "author": "Kayo & Fox Sports Web Developers",
+  "author": "Streamotion Web Developers",
+  "repository": {
+    "type": "git",
+    "url": "ssh://git@github.com:FoxSportsAustralia/eslint-config-foxsports.git"
```

Comment:
You will need to move this repo to FSA-Streamotion space to publish. Chat with Nesa/Szasza about it.